pkg/errors/sdk_errors.go

```diff
@@ -79,17 +77,19 @@ func CheckError(resp *http.Response, expectedStatusCodes ...int) SDKError {
 		}
 	}
 
-	var content map[string]interface{}
-	if err := json.NewDecoder(resp.Body).Decode(&content); err != nil {
-		return NewSDKErrorWithStatus(err, resp.StatusCode)
+	body, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return NewSDKErrorWithStatus(fmt.Errorf("%w : %w", errRespBody, err), resp.StatusCode)
```

Comment:
Try removing the space before the colon to be sth like  `%w: %w`. This could be the reason why codecov is failing

Reply:
@ianmuchyri 
Error is due to using of two error wrapping in `Errorf`

Reply:
Resloved